src/configs/orm.config.ts

```diff
@@ -1,19 +1,40 @@
 import { registerAs } from '@nestjs/config';
 import { DataSource, DataSourceOptions } from 'typeorm';
 import { env } from './env.config';
+import { CreateRoleTable1726916715779 } from '~migrations/1726916715779-CreateRoleTable';
+import { CreateUserTable1726975405484 } from '~migrations/1726975405484-CreateUserTable';
+import { CreateUserResetPasswordTable1727517927086 } from '~migrations/1727517927086-CreateUserResetPasswordTable';
+import { CreateCarTable1729394157614 } from '~migrations/1729394157614-CreateCarTable';
+import { CreateOrderTable1729395014155 } from '~migrations/1729395014155-CreateOrderTable';
+import { Car } from '~modules/cars/car.entity';
+import { Order } from '~modules/orders/order.entity';
+import { Role } from '~modules/users/entities/role.entity';
+import { UserResetPassword } from '~modules/users/entities/user-reset-password.entity';
+import { User } from '~modules/users/entities/user.entity';
 
-const config = {
+const config: DataSourceOptions = {
   type: 'postgres',
   host: env.DB_HOST, // Hoặc giá trị host thực tế
   port: Number(env.DB_PORT),
   username: env.DB_USERNAME, // Thay bằng username PostgreSQL của bạn
   password: env.DB_PASSWORD, // Thay bằng password của bạn
   database: env.DB_NAME, // Thay bằng tên database bạn sử dụng
-  entities: env.ENTITIES,
-  migrations: env.MIGRATIONS,
-  autoLoadEntities: true,
+  entities: [Role, User, UserResetPassword, Car, Order],
+  migrations: [
+    CreateCarTable1729394157614,
+    CreateUserTable1726975405484,
+    CreateRoleTable1726916715779,
+    CreateOrderTable1729395014155,
+    CreateUserResetPasswordTable1727517927086,
+  ],
```

Comment:
This way is to avoid this error, wrong path
![image](https://github.com/user-attachments/assets/a8d7de0c-e05c-435d-89f1-94d8463e5c45)